### PR TITLE
Additions and Tweaks

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request
.45 speedloader replaced with .38 speed loader in trash piles, due to the relative rarity of .45 revolvers
Chinese outfits added to costume loot, for people who want to RP as shi
Flak Helmet added to head loadout, because the flak jacket not having it's pair was annoying me to no end, and slightly buffed
Flak jacket brought to parity with the military jacket

## Why It's Good For The Game

every other wastelander has the detective revolver and police revolvers exist and while not overly good are the only readily available boot guns. makes more sense for their ammo to be more available
flak helmets make sense to go with the flak jacket, it doesnt spawn anywhere on map, and all other helmets are faction based (steel pot) or have futuristic visors on them (combat helmet)
qipao and cheongsam were totally nuked with the removal of it in loadout and the clothesmate, at least have it be available in lockers
flak vest is now on par with a piece of heavy clothing, and has its loadout cost increased as well

## Changelog
:cl:
add: flak helmet to loadout
add: cheongsam and qiapo
tweak: flak vest slightly
tweak: .38s instead of .45s in trashpiles
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -968,13 +968,13 @@
 
 /obj/item/clothing/suit/flakjack
 	name = "flak jacket"
-	desc = "(I) A dilapidated jacket made of a supposedly bullet-proof material (Hint: It isn't.). Smells faintly of napalm."
+	desc = "(I) A dilapidated jacket made of ballistic nylon. Smells faintly of napalm."
```
Comment (Pebblesthe2nd) [inline code/modules/clothing/suits/miscellaneous.dm:971]:
```suggestion
	desc = "(II) A dilapidated jacket made of ballistic nylon. Smells faintly of napalm."
```

The description should always match the coded tier value. Armor undergoing a refactoring anyways but while we still have linearmor thats the format.

**Review by Pebblesthe2nd — CHANGES_REQUESTED** [review]
Shi sounds like some Bethesda stuff which is not canon for the server. A couple articles of chinese clothing in the prewar clothing spawner seems harmless though.  .38 being more common than .45 makes sense, and the flak stuff is sensible. With the correct desc on the flak jacket it looks good.

**Comment by cornercube** [comment]
> Shi sounds like some Bethesda stuff which is not canon for the server. A couple articles of chinese clothing in the prewar clothing spawner seems harmless though. .38 being more common than .45 makes sense, and the flak stuff is sensible. With the correct desc on the flak jacket it looks good.

shi is from fallout 2, although to be fair, black isle said they wanted them to be nuked in NV
flak jacket updated to II in desc.

**Comment by Rohesie** [comment]
Code-wise it just requires this little change.

**Review by Pebblesthe2nd — APPROVED** [review]

**Pull request merged**